float64[]   actual_traversal_costs
float64[]   policy_output_costs
